### ISSUE TITLE
Problems in reproduce TSP100

### ISSUE THREAD

**Issue by Layphone** [yimengmin/UTSP#4]
Hello, Yimeng

I have some trouble in reproducing the TSP100. 

I used the command to train and got a final value of 39 for the loss function：
python train.py --num_of_nodes 100 --EPOCHS 300 --batch_size 32 --temperature 3.5 --C1_penalty 20.0 --nlayers 2 --hidden 64 --rescale 2.0 --moment 1 --lr 5e-3 --stepsize 20

Then I tried to Generate TSP 100 heatmaps (test file) and used the command: 
python loadmodel.py --num_of_nodes 100 --batch_size 128 --temperature 3.5 --nlayers 2 --hidden 64 --rescale 2.0 --moment 1

However, the result is : It takes 1.19157 seconds from instance: 0 to 128......It takes 0.02299 seconds from instance: 9984 to 10000. And the code doesn't seem to stop.
![微信图片_20240229192844](https://github.com/yimengmin/UTSP/assets/152859167/26b64860-7916-48b5-a4f4-4fba977e8b7a)

Best regards,
Zhuohui

**Comment by yimengmin** [comment]
It looks like you pressed Control+C, which caused it to stop?

also, check the new readme file,  which provides instructions on how to train and search TSP-100.

**Comment by Layphone** [comment]
Hello Yimeng,

Thanks for your reminding, I just found that you have updated the file.I followed your instructions and the operation succeeded. 

Now I have another question about the output results after searching：the final optimal solution is  average of all **Avg_MCTS_Distance** values in the result files, isn't it
![微信图片_20240304141426](https://github.com/yimengmin/UTSP/assets/152859167/670140a1-00d5-4086-a080-9f80eb538a17)

**Comment by yimengmin** [comment]
yes, there are gap (%) after it, and the real distance is the reported distance / rescale .
